build-tools/packages/build-cli/.prettierignore

```diff
@@ -1,5 +1,6 @@
 /api-report
 /dist
```

Comment:
Do we still need to include "dist" in the various ignore files?

Reply:
Left these in intentionally because existing repos will have the dist files already, and I don't want to risk them getting checked in.